core/src/main/java/org/apache/struts2/dispatcher/Dispatcher.java

```diff
@@ -340,6 +348,27 @@ public void setMultipartValidationRegex(String multipartValidationRegex) {
         this.multipartValidationPattern = Pattern.compile(multipartValidationRegex);
     }
 
+    @Inject(value = StrutsConstants.STRUTS_ACTION_EXCLUDE_PATTERN_SEPARATOR, required = false)
+    public void setActionExcludedPatternsSeparator(String separator) {
+        this.actionExcludedPatternsSeparator = separator;
+    }
+
+    @Inject(value = StrutsConstants.STRUTS_ACTION_EXCLUDE_PATTERN, required = false)
+    public void setActionExcludedPatterns(String excludedPatterns) {
+        this.actionExcludedPatterns = buildExcludedPatternsList(excludedPatterns, actionExcludedPatternsSeparator);
+    }
+
+    private static List<Pattern> buildExcludedPatternsList(String patterns, String separator) {
+        if (patterns == null || patterns.trim().isEmpty()) {
+            return emptyList();
+        }
+        return unmodifiableList(Arrays.stream(patterns.split(separator)).map(String::trim).map(Pattern::compile).collect(toList()));
+    }
+
+    public List<Pattern> getActionExcludedPatterns() {
+        return actionExcludedPatterns;
+    }
```

Comment:
It's getting crowd'y here, what about moving all the injectable options into `DispatcherOptions`? It doesn't have to happen now, but I can create a ticket to address that later

Reply:
Yep I think that's a fair call, I'll create an issue for it

Reply:
WW-5410